### ISSUE TITLE
Chain-of-Thought Prompting Elicits Reasoning in Large Language Models

### ISSUE THREAD

**Issue by reyllama** [reyllama/paper-reviews#225]
# TL; DR

- Prompting an LLM with *chain-of-thought* reasoning greatly improves complex multi-step problem solving (e.g., math).

# Motivation

- Prompting matters in LMs.

# Formulation

<img width="925" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/1cba43ea-99a5-43ba-8df4-6f8dc51f3f0c">
<img width="709" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/a0a56c23-6902-4258-9f66-4b05d6351f02">

# Evaluation

- **1. Chain-of-thought scales with model size.**
- **2. Chain-of-thought scales with task complexity.**
- **3. Chain-of-thought prompting on GPT-3 or PaLM-540B achieves state-of-the-art.**

<img width="395" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/b6a6866c-401b-4aca-92bd-d7db70f75ce1">
<img width="365" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/040619d8-fdfe-4a56-a2c1-7c5092652ec7">

<img width="261" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/3e024116-6794-481f-a0c2-381689b4cfdf">
<img width="292" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/8374e156-b69d-4e78-bbe2-cb5465b37672">

- The linguistic semantics and sequential reasoning both matter.
- Chain-of-thought is relatively robust across specific prompt styles/formats.

<img width="695" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/373abec6-b467-4f12-b351-dbc3861c4883">
<img width="295" alt="image" src="https://github.com/reyllama/paper-reviews/assets/46308407/3c88b468-5141-4719-b014-c0ab1918610e">

# Inspiration

- Let's think this step-by-step did indeed really work :)
